Make group views (new, edit)

miya98935271 opened the pull request:
# What
groupモデルのnew、editのviewを作る

# Why
chat-view内で新しいチャットグループを作成、編集できるようにするため

Commit:
Make group view (new, edit)

- Make new-view file of group
- Make edit-view file of group
- Set routing for maked views

Commit:
Update .DS_Store

miyagit commented on app/views/groups/new.html.haml:
```diff
@@ -0,0 +1,20 @@
+.chat-group-form
```
こちら部分テンプレートを使いましょう。下記のサイトに使い方詳しく載ってるので、下記を参考に実装しましょう。
https://www.pikawaka.com/rails/partial_template

miya98935271 replied:
フォーム部分を部分テンプレート化しました。
合わせてmessages/indexのeditボタンの属性を修正しました。
(データを送信しないのでsubmitからbuttonにtypeを変更しています。)

miyagit reviewed (commented):
LGTM

Commit:
Make partial file

- Make partial file for group view
- Correct  type element of "edit" button
 submit into button

shingoteshima commented:
LGTMです！
